wordpress-mobile/gutenberg-mobile
# Fix Enter on post title.

Fixes #1053 

GB PR: https://github.com/WordPress/gutenberg/pull/15944

To test:
 - Start the demo app
 - Tap on the title 
 - Write some content
 - Press Enter
 - Check that a new paragraph block is added after the title and it's focused.

Update release notes:
